tests/unit/amo/components/TestAutoSearchInput.js

```diff
@@ -108,30 +114,33 @@ describe(__filename, () => {
 
     it('renders an initial query', () => {
       const query = 'ad blocker';
-      const root = render({ query });
+      const location = createFakeLocation({ query: { query } });
+
+      const root = render({ location });
 
       expect(getInputProps(root)).toMatchObject({ value: query });
     });
 
-    it('renders an updated query', () => {
-      const root = render();
+    it('sets the search value to an empty string when there is no `location.query`', () => {
+      const location = createFakeLocation({ query: null });
 
-      const query = 'ad blocker';
-      root.setProps({ query });
+      const root = render({ location });
 
-      expect(getInputProps(root)).toMatchObject({ value: query });
+      expect(root).toHaveState('searchValue', '');
     });
 
     it('only sets an updated query if it is unique', () => {
```

Comment:
This spec could use an update. I'm not sure if it even made sense with the old code 😛